Search bar

TheoA816 opened the pull request:
Lies not just the search bar.

- Search Bar which allows you to filter events by societies
- Tag List to filter event by tags (Networking, Sport etc.)
- Edited overall FE here and there

Suchpuns reviewed (changes requested):
# LOOKS GREAT!!! :D

## One last thing...
the search bar is a little wonky still.
When you hover over a dropdown it goes bigger and looks a little weird.
![image](https://user-images.githubusercontent.com/54874091/193388798-e8322138-607d-4616-b0e0-1fb22fa76c06.png)

Also, can you make the search bar dropdown items change color as well if that society is selected/unselected? Also, you can't unselect societies with the dropdown.

Keep up the great work!

TheoA816 commented:
shud be all goods now

i removed the getting bigger thing altogether but you can select and unselect from the search bar now. And the weird onhover thing that happened for the societies was also happening for the tags so i fixed that up

Suchpuns reviewed (approved):
LGTM!!!!!
